FpgaOverlayManager.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-const SCRIPT_PATH = "../utils"
+const util = require('./utilities.js');
+
+const SCRIPT_PATH = "../utils/runtime_config"
```

Comment:
Just a note to self here that this now requires cloning our utils repo, whereas before I was copying the runtime_config folder from the repo to a utils folder on the device.

Reply:
Ah, I wasn't really sure what you were doing before. It just seemed more straightforward to just use the repo directly rather than needing to clone it somewhere else and then only copy the runtime_config into a utils directory.

Reply:
I agree cloning the repo is more straightforward. Ultimately we'll make some sort of "installer" or at least copy our files into the rootfs when we build it.